docs/data/sdks/typescript-browser/migration.md

```diff
@@ -0,0 +1,497 @@
+  identify
+    .set("membership", "paid")
+    .set("payment", "bank")
+  amplitude.identify(identify)
+  ```
+
+#### `identify()`
+
+Making an identify call can be done on `amplitude` without calling `getInstance()`.
+
+=== "amplitude-js"
+  ```javascript
+  const identify = new amplitude.Identify()
+  identify.set("membership", "paid")
+  amplitude.getInstance().identify(identify)
+  ```
+
+=== "@amplitude/analytics-browser"
+  ```typescript
+  const identify = new amplitude.Identify()
+  identify.set("membership", "paid")
+  amplitude.identify(identify)
+  ```
+
+### Setting group properties
+
+### `groupIdentify()`
+
+Making an identify call can be done on `amplitude` without calling `getInstance()`.
+
+=== "amplitude-js"
+  ```javascript
+  const identify = new amplitude.Identify()
+  identify.set("membership", "paid")
+  amplitude.getInstance().groupIdentify(identify)
+  ```
+
+=== "@amplitude/analytics-browser"
+  ```typescript
+  const identify = new amplitude.Identify()
+  identify.set("membership", "paid")
+  amplitude.groupIdentify(identify)
+  ```
+
+### Tracking revenue
+
+#### `logRevenueV2()`
+
+Tracking revenue can be done using `revenue()` API on `amplitude` without calling `getInstance()`.
+
+=== "amplitude-js"
+  ```javascript
+  const revenue = new amplitude.Revenue()
+  revenue
+    .setProductId("productId")
+    .setPrice(10)
+  amplitude.getInstance().logRevenueV2(revenue)
+  ```
+
+=== "@amplitude/analytics-browser"
+  ```typescript
+  const revenue = new amplitude.Revenue()
+  revenue
+    .setProductId("productId")
+    .setPrice(10)
+  amplitude.revenue(revenue)
+  ```
+
+### Patterns
+
+#### Plugins
+
+The configs `config.language`, `config.library`, `config.platform` were added to `amplitude-js` to allow modification of event payloads for these specific fields. While `@amplitude/analytics-browser` do not support these configurations anymore, the new Browser SDK allows for plugins to be added and enrich event payloads.
+
+=== "@amplitude/analytics-browser"
+  ```ts
+  import { BrowserConfig, EnrichmentPlugin, Event, PluginType } from '@amplitude/analytics-types'
+
+  export class LibraryModifierPlugin implements EnrichmentPlugin {
+    name = 'library-modifier'
+    type = PluginType.ENRICHMENT as const
+
+    /**
+    * setup() is called on plugin installation
+    * example: client.add(new LibraryModifierPlugin());
+    */
+    setup(config: BrowserConfig): Promise<undefined> {
+      this.config = config
+    }
+
+    /**
+    * execute() is called on each event instrumented
+    * example: client.track('New Event');
+    */
+    execute(event: Event): Promise<Event> {
+      event.library = 'my-library-name/1.0.0'
+      return Promise.resolve(event)
+    }
+  }
+  ```
+
+#### Defer initialization
+
+To defer initialization in `amplitude-js`, init must be called with `config.deferInitialization` set to true, and eventually calling `enableTracking()` to formalize initialization and send all enqueued events.
```

Comment:
I think we should follow the same pattern. One paragraph to mention the issue. and then one code snippet with 2 language.